src/gsy_e/models/strategy/predefined_pv.py

```diff
@@ -90,7 +90,7 @@ def set_produced_energy_forecast_in_state(
 
     def reconfigure(self, **kwargs):
         """Reconfigure the device properties at runtime using the provided arguments."""
-        if key_in_dict_and_not_none(kwargs, "cloud_coverage"):
+        if "cloud_coverage" in kwargs:
```

Comment:
We can only do this, because we validate the kwargs before (in the json schema validator) and check if `cloud_coverage ` is not None, right?